corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -473,6 +473,34 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         },
     });
 
+    const CaseTileViewUnclickable = CaseTileView.extend({
```

Comment:
Seems like the inheritance should be the other way around since `CaseTileView` has more functionality but I that would probably be tricky.

Reply:
I think this makes sense, so I played around with it, but now I'm inclined to leave it as-is. I did a commit that made `CaseViewUnclickable` the parent view of `CaseView`, which is fine. `CaseTileViewUnclickable` can then be a child of `CaseViewUnclickable`, but `CaseTileView` needs functionality from both `CaseView` and `CaseTileViewUnclickable` which gets messy.